docs/docs/basics/01-tx-lifecycle.md

```diff
@@ -157,37 +157,41 @@ must be in this proposer's mempool.
 ## State Changes
 
 The next step of consensus is to execute the transactions to fully validate them. All full-nodes
-that receive a block proposal from the correct proposer execute the transactions by calling the ABCI functions
-[`BeginBlock`](./00-app-anatomy.md#beginblocker-and-endblocker), `DeliverTx` for each transaction,
-and [`EndBlock`](./00-app-anatomy.md#beginblocker-and-endblocker). While each full-node runs everything
-locally, this process yields a single, unambiguous result, since the messages' state transitions are deterministic and transactions are
+that receive a block proposal from the correct proposer execute the transactions by calling the ABCI function `FinalizeBlock`. 
+While each full-node runs everything locally, this process yields a single, unambiguous result, 
+since the messages' state transitions are deterministic and transactions are
 explicitly ordered in the block proposal.
 
 ```text
 		-----------------------
 		|Receive Block Proposal|
 		-----------------------
+							|
+				v
+		-------------------------
+		| FinalizeBlock	        |
 		          |
 			  v
-		-----------------------
-		| BeginBlock	      |
-		-----------------------
+				-------------------
```

Comment:
can we get the pretty alignment back?

Reply:
do you know an extension to do this?

Reply:
This diagram will suffice for now but I think we should get the icf designer to do a fancy version wyt?

Reply:
im good with that